iis-workflows/iis-workflows-top/src/main/resources/eu/dnetlib/iis/workflows/top/common/import/job.properties

```diff
@@ -12,21 +12,18 @@ project_concepts_context_id=fet-fp7
 database_service_location=http://beta.services.openaire.eu:8280/is/services/database
```

Comment:
I will fix this by putting `job.properties` to `.gitignore`, already reported in different pull request.